model/src/main/scala/org/broadinstitute/dsde/rawls/model/WorkspaceModel.scala

```diff
@@ -577,11 +577,11 @@ case class WorkspaceDetails(namespace: String,
                             isLocked: Boolean = false,
                             authorizationDomain: Option[Set[ManagedGroupRef]],
                             workspaceVersion: WorkspaceVersion,
-                            googleProjectId: GoogleProjectId,
+                            googleProject: GoogleProjectId, // The response field is called "googleProject" rather than "googleProjectId" for backwards compatibility
                             googleProjectNumber: Option[GoogleProjectNumber],
                             billingAccount: Option[RawlsBillingAccountName],
-                            billingAccountErrorMessage: Option[String]) {
-  def toWorkspace: Workspace = Workspace(namespace, name, workspaceId, bucketName, workflowCollectionName, createdDate, lastModified, createdBy, attributes.getOrElse(Map()), isLocked, workspaceVersion, googleProjectId, googleProjectNumber, billingAccount, billingAccountErrorMessage) // todo: do we just add the error message on here? what does that mean for the api?
+                            billingAccountErrorMessage: Option[String] = None) {
```

Comment:
Just want to confirm that this default value change was intentional in the PR?  Since all the other changes seemed to be about `googleProjectId` -> `googleProject`

Reply:
Yup, this was intentional. I don't recall exactly what I was trying to fix, but I was fixing something. It also logically makes sense to me for the error message to default to None